notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>Index 19? Do you mean index 1 or 11? Likely 1 based on the next cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Yep, definitely meant 1. I realized that the "2 of our 17" could possibly change in the future? Are more processed light curves being added to QLP and TASOC? I added "(as of the writing of this tutorial)" to the end of that sentence. However, I don't <em>think</em> my choice of index 1 would change in the future becuase it's based on the Vizier table  and the order shouldn't even change. Should I hard code my choice of RA and Dec instead?</p>

Reply:
<p>Yes, QLP and TASOC both deliver new data frequently, so there's a chance that there will be more targets with light curves in the future. I think adding "as of the writing of this tutorial" is a good fix for the first sentence, but I would specify the RA/Dec rather than the index number.</p>